app/src/pages/admin/pluginsPage/pluginsItems/pluginsItems.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { fetchPluginsAction, pluginsSelector } from 'controllers/plugins';
+import { AUTHORIZATION_GROUP_TYPE } from 'common/constants/pluginsGroupTypes';
+import { PageLayout, PageHeader, PageSection } from 'layouts/pageLayout';
+import { PluginsTabs } from './../pluginsTabs';
+
+const messages = defineMessages({
+  pageTitle: {
+    id: 'PluginsPage.title',
+    defaultMessage: 'Plugins',
+  },
+});
+
+@connect(
+  (state) => ({
+    plugins: pluginsSelector(state),
+  }),
+  {
+    fetchPluginsAction,
+  },
+)
+@injectIntl
+export class PluginsItems extends Component {
```

Comment:
Seems that this `PluginsItems` class very similar with `PluginsPage` and it never used. Please leave the one of them.